api/client/transform/fortio.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package nighthawk.client;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+import "validate/validate.proto";
+
+message FortioResult {
+  // Disable validate because it doesn't work well with fields that are not all lowercase.
+  // The field cases must match the final output JSON that fortio-ui expects.
```

Comment:
Please put a top-level comment explaining the proto3-JSON trick that is being carried out here.

Reply:
Done.